test_suite/cloud/test_aws.py

```diff
@@ -570,13 +571,9 @@ def test_yum_group_install(self, host):
                         ' && yum clean all'
                     )
 
-                    result_second_attempt = test_lib.print_host_command_output(
-                        host, dev_tools_install_command, capture_result=True
-                    )
-
-                    assert result_second_attempt.succeeded, (
-                        'Error while installing Development tools group after two attempts. '
-                        'Check test case output for more details.'
+                    assert host.run(dev_tools_install_command).succeeded, (
+                        f'Error while installing Development tools'
+                        f' group after two attempts with error: {result.stderr}'
                     )
                     print('"Development tools" installed successfully.')
```

Comment:
This print should be indented outside the two `if` statements.

Reply:
ack